draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -162,18 +158,22 @@
 
 	  <t hangText="Issuer">
 	    <vspace/>
-	    Party that creates the CWT and binds its claims to the proof-of-possession key.
+		Party that creates the CWT and binds the claims about the subject to the 
+		proof-of-possession key.
 	  </t>
 
 	  <t hangText="Presenter">
 	    <vspace/>
 	    Party that proves possession of a private key (for asymmetric key cryptography)
-	    or secret key (for symmetric key cryptography) to a recipient.
+	    or secret key (for symmetric key cryptography) to a recipient.<vspace/>
+		In context of OAuth this party is also called the OAuth Client.
```

Comment:
Please delete the <vspace/> as it will cause an odd line break between sentences.  Put a comma after OAuth.